### ISSUE TITLE
[hw4] Question about Json formating

### ISSUE THREAD

**Issue by tonypark7277** [swsnu/swppfall2021#132]
In README.md, it says that
"Also, please make sure to implement your request methods under the following detailed specifications (in your views.py). When the server successfully handles a request, it responds with a JSONResponse for GET(except /api/signout/) and responds with a HTTPResponse for POST, PUT, and DELETE."

Is it okay to use JSONResponse for POST, PUT, or DELETE method?

**Comment by ttoru96** [comment]
Yes, it is okay to use JSONResponse for POST, PUT, or DELETE method. 

Please refer to https://github.com/swsnu/swppfall2021/issues/113#issuecomment-951820555 for more detailed explanation.

**Comment by tonypark7277** [comment]
Thank you for quick reply! I solve my another problem thanks to your additional advice
